Spring.FluentContext/ContextRegistrations.cs

```diff
@@ -0,0 +1,210 @@
+using System.Reflection;
+using Spring.Context;
+using Spring.Context.Support;
+using Spring.FluentContext.BuildingStages.Aliases;
+using Spring.FluentContext.BuildingStages.Objects;
+using Spring.FluentContext.BuildingStages.ProxyFactories;
+using Spring.FluentContext.Definitions;
+using Spring.FluentContext.Impl;
+using Spring.FluentContext.Utils;
+using Spring.Objects.Factory.Config;
+using Spring.Validation;
+using Spring.Validation.Actions;
+
+namespace Spring.FluentContext
+{
+    public static class ContextRegistrations
```

Comment:
I have to think more about this class. It drops the convention used in the library where you access those methods by `IFluentApplicationContext` and interfaces it implements. At the same time I understand that while the `FluentApplicationContext` was the only context configurable fluently, now you want to make all contexts to be configurable this way. Will come back to this.